tests/www/test_views.py

```diff
@@ -95,7 +95,7 @@ def clear_table(cls, model):
 
     def check_content_in_response(self, text, resp, resp_code=200):
         resp_html = resp.data.decode('utf-8')
-        self.assertEqual(resp_code, resp.status_code)
+        assert resp_code == resp.status_code, str(resp_html)
```

Comment:
Can you use assertions from [unittest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual) package?  Currently, this convention is used throughout the entire project. Keeping one style in the whole project facilitates its development.